lib/dokumi/support/xml.rb

```diff
@@ -0,0 +1,17 @@
+module Dokumi
+  module Support
+    module XML
+      def self.read_xml_tag(text)
+        h = {}
+        s = StringScanner.new(text)
```

Comment:
This implementation is already exist and I just moved from the another module.
I think we should refactor it in other PR.